src/pages/benchmarks/index.tsx

```diff
@@ -12,21 +12,50 @@ import $button from "../../components/design_system/button/Button.module.css";
 import noBenchmarks from "../../public/img/no-benchmarks.png";
 import SearchIcon from "@mui/icons-material/Search";
 
+type SortProperty = "first_name";
+type SortDirection = "asc" | "desc";
+
 function Benchmarks() {
   const [isPara, setIsPara] = useState(false);
-  const { data: tasks, isLoading } = trpc.para.getMyTasks.useQuery();
+
+  const [sortProperty, setSortProperty] = useState<SortProperty>("first_name");
+  const [sortDirection, setSortDirection] = useState<SortDirection>("asc");
+
+  const { data: tasksData, isLoading, error } = trpc.para.getMyTasks.useQuery();
 
   const handleTogglePara = () => {
     setIsPara(!isPara);
   };
 
+  const getSortedTasks = () => {
+    if (!tasksData) return [];
+    return [...tasksData].sort((a, b) => {
+      if (a[sortProperty] < b[sortProperty])
+        return sortDirection === "asc" ? -1 : 1;
```

Comment:
I had a bit of trouble following why you want to return 1 or -1. Could you please explain? Thanks!

Reply:
JavaScript decides whether two items in a sort are in the correct order relative to another depending on whether the sorting callback you give it returns +1 or -1.

Reply:
ty!